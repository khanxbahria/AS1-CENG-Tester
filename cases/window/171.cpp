#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->getActiveTab();
Tab E = Tab("cc", "dd","kkk");
wind->changeActiveTabTo(4);
Tab O = Tab("cc", "nn","y");
wind->newTab(E);
Tab F = Tab("l", "l","mmm");
wind->isEmpty();
Tab V = Tab("zz", "zzz","p");
wind->moveActiveTabTo(1);
Tab P = Tab("iii", "kk","uuu");
wind->changeActiveTabTo(2);
Tab J = Tab("ss", "u","mmm");
wind->isEmpty();
Tab A = Tab("www", "bb","x");
wind->closeTab();
wind->closeTab();
Tab Z = Tab("hhh", "ii","ff");
wind->getActiveTab();
Tab X = Tab("bbb", "cc","uu");
wind->print();
Tab L = Tab("fff", "ttt","ooo");
wind->closeTab();
Tab M = Tab("jjj", "n","dd");
wind->newTab(A);
Tab H = Tab("dd", "vvv","a");
wind->getActiveTab();
Tab G = Tab("fff", "u","l");
wind->isEmpty();
wind->print();
wind->moveActiveTabTo(3);
Tab N = Tab("ddd", "aaa","aaa");
wind->changeActiveTabTo(3);
wind->changeActiveTabTo(0);
wind->newTab(F);
wind->print();
Tab D = Tab("ccc", "mmm","ii");
wind->newTab(D);
Tab K = Tab("qqq", "yy","w");
wind->moveActiveTabTo(0);
wind->closeTab();
wind->closeTab();
Tab C = Tab("mmm", "mm","sss");
wind->moveActiveTabTo(3);
wind->newTab(C);
wind->print();
Tab T = Tab("zzz", "f","n");
wind->moveActiveTabTo(0);
wind->newTab(G);
wind->getActiveTab();
wind->moveActiveTabTo(5);
Tab W = Tab("b", "ddd","aaa");
wind->getActiveTab();
wind->isEmpty();
wind->getActiveTab();
wind->isEmpty();
wind->closeTab();
Tab U = Tab("ccc", "jj","h");
wind->print();
wind->getActiveTab();
wind->isEmpty();
wind->closeTab();
Tab Q = Tab("c", "yy","j");
wind->moveActiveTabTo(1);
wind->isEmpty();
wind->newTab(H);
wind->isEmpty();
wind->print();
wind->changeActiveTabTo(4);
wind->newTab(J);
Tab S = Tab("t", "zzz","oo");
wind->getActiveTab();
wind->isEmpty();
wind->getActiveTab();
Tab R = Tab("qqq", "kkk","xx");
wind->newTab(K);
wind->newTab(L);
wind->closeTab();
wind->closeTab();
wind->isEmpty();
wind->print();
Tab B = Tab("h", "bbb","k");
wind->changeActiveTabTo(4);
wind->isEmpty();
wind->closeTab();
wind->newTab(B);
wind->isEmpty();
wind->newTab(M);
wind->isEmpty();
wind->isEmpty();
wind->print();
wind->print();
wind->newTab(N);
wind->moveActiveTabTo(2);
wind->newTab(O);
wind->print();
wind->changeActiveTabTo(3);
wind->getActiveTab();
wind->changeActiveTabTo(0);
wind->newTab(P);
wind->changeActiveTabTo(0);
wind->closeTab();
wind->closeTab();
wind->moveActiveTabTo(0);
wind->closeTab();
wind->getActiveTab();
wind->isEmpty();
wind->print();
wind->moveActiveTabTo(5);
wind->isEmpty();
wind->getActiveTab();
wind->isEmpty();
wind->closeTab();
Tab Y = Tab("ccc", "nnn","s");
wind->getActiveTab();
wind->getActiveTab();
wind->print();
wind->getActiveTab();
wind->getActiveTab();
wind->isEmpty();
wind->isEmpty();
wind->print();
wind->moveActiveTabTo(0);
wind->closeTab();
wind->print();
wind->newTab(Q);
delete wind;
}
	return 0;
}