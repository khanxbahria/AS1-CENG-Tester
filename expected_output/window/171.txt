Window is empty
The list is empty.
The active tab is tab 0
bb
www
x

The active tab is tab 1
bb
www
x

l
l
mmm

The active tab is tab 1
l
l
mmm

mm
mmm
sss

The active tab is tab 1
mm
mmm
sss

l
l
mmm

The active tab is tab 1
mm
mmm
sss

vvv
dd
a

The active tab is tab 2
mm
mmm
sss

vvv
dd
a

u
ss
mmm

The active tab is tab 3
mm
mmm
sss

vvv
dd
a

bbb
h
k

n
jjj
dd

The active tab is tab 3
mm
mmm
sss

vvv
dd
a

bbb
h
k

n
jjj
dd

The active tab is tab 3
mm
mmm
sss

vvv
dd
a

aaa
ddd
aaa

nn
cc
y

bbb
h
k

n
jjj
dd

The active tab is tab 0
aaa
ddd
aaa

nn
cc
y

bbb
h
k

n
jjj
dd

The active tab is tab 2
nn
cc
y

bbb
h
k

n
jjj
dd

The active tab is tab 2
nn
cc
y

bbb
h
k

n
jjj
dd

The active tab is tab 0
nn
cc
y

bbb
h
k

